app/partials/referencedByView.html

```diff
@@ -0,0 +1,30 @@
+<record-address-bar ng-if="showAddressBar"></record-address-bar>
+<!-- TODO: search bar disabled for alpha because not coded yet -->
+<search-bar ng-if="false" ng-controller="recordSearchController"></search-bar>
+<div ng-if="record" class="column-wrapper">
+    <div class="tree-columns">
+        <span ng-repeat-start="column in columns" class="column column-{{::$index}}" ng-class="{'sortable' : column.canSort}" ng-click="toggleSort(column)">
+            {{::column.label}}
+            <span class="sort-arrow" ng-if="column.label === sort.column" title="Elements are sorted by this column in {{sort.reverse ? 'ascending' : 'descending'}} order.">
+                <i class="fa fa-arrow-{{sort.reverse ? 'up' : 'down'}}"></i>
+            </span>
+        </span>
+        <split-bar ng-repeat-end ng-if="!$last" offset="16" resize-callback="columnResized"></split-bar>
+        <div class="column-action edit-columns" ng-click="openColumnsModal()" title="Edit columns"></div>
+    </div>
+</div>
+<div class="tree-nodes" tabindex="0" ng-keydown="onTreeKeyDown($event)" ng-mousedown="onTreeMouseDown($event)" theme-scrollbar-fix vs-repeat>
+    <div ng-repeat="node in tree" ng-mousedown="onNodeMouseDown($event, node)" ng-dblclick="onNodeDoubleClick($event, node)" ng-drag="onNodeDrag(node)" ng-class="{'selected' : node.selected}" class="node {{node.class}}" ng-init="node.has_data || getNodeData(node)">
```

Comment:
creating a whole "tree view" in the referenced by view seems kind of overkill to me.  If we don't have nodes that expand or multiple types of nodes we should be able to use a simpler implementation.  What I'm suggesting won't change the HTML much, but it will change the JavaScript for sure.

Reply:
I would like to not use "tree" in classes, functions, or scope variables for this view, because it's not a tree.